### PR TITLE
fix: generate go binding for IndexRegistry

### LINKED ISSUE #200: Missing bindings for IndexRegistry
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by renlulu**
Fixes #200  .

### Motivation
Sometimes we need get operator list from IndexRegistry for some purpose, but realize that sdk doesn't contain bindings for IndexRegistry contract right now.

### Solution
Modify `generate-bindings.sh` a bit and ask it generate everything for us : D

### CODE REVIEW

**Comment by samlaf** [comment]
Thanks! We hadn't needed it so far but should absolutely add this.
We should also add it to the bindings util: https://github.com/Layr-Labs/eigensdk-go/blob/84d5bc10f72f15f68387de116bfbb8297258e017/chainio/utils/bindings.go#L86
Would you mind adding it here as well?

**Comment by renlulu** [comment]
> AvsRegistryContractBindings

Yes sure!

**Review by NimaVaziri — APPROVED** [review]

**Review by NimaVaziri — APPROVED** [review]

**Comment by bb111189** [comment]
thanks folks for the speedy review - junhao

File: chainio/utils/bindings.go [hunk 1]
```diff
@@ -90,6 +90,7 @@ type AvsRegistryContractBindings struct {
 	StakeRegistryAddr          gethcommon.Address
 	BlsApkRegistryAddr         gethcommon.Address
 	OperatorStateRetrieverAddr gethcommon.Address
+	IndexRegistryAddr          gethcommon.Address
```
Comment (samlaf) [inline chainio/utils/bindings.go:93]: You only added the addr. You need to also add the bindings for the IndexRegistry.

Comment (renlulu) [reply chainio/utils/bindings.go:93]: Oh yes, just added, sorry

**Review by samlaf — APPROVED** [review]
LGTM. will merge if tests pass

**Pull request merged**